src/main/scala/org/hammerlab/guacamole/commands/SomaticStandardCaller.scala

```diff
@@ -113,6 +117,28 @@ object SomaticStandard {
           args.minTumorAlternateReadDepth
         )
 
+      if (args.dbSnpVcf != "") {
+        val adamContext = new ADAMContext(sc)
+        val dbSnpVariants = adamContext.loadVariantAnnotations(args.dbSnpVcf)
+        potentialGenotypes
+          .keyBy(_.adamVariant)
+          .leftOuterJoin(dbSnpVariants.keyBy(_.variant))
+          .map(_._2).map({
+            case (calledAllele: CalledSomaticAllele, dbSnpVariant: Option[DatabaseVariantAnnotation]) =>
+              CalledSomaticAllele(
+                sampleName = calledAllele.sampleName,
```

Comment:
any reason not to use `calledAllele.copy(rsID = dbSnpVariant.map(_.getDbSnpId))` here?